### ISSUE TITLE
Drop-down menu is not populated with serial ports list on launch (MacOS)

### ISSUE THREAD

**Issue by federicoVisi** [wearablecomputing/flash#2]
Hi Ben, I tested the new release on Mac. Unfortunately the drop-down menu does not get populated when you open the Max patch. After displaying "Updating..." on the button for a few minutes I got "flash: Timeout. Update process took longer than expected." on both Max console and message box at the bottom of the patch. 

Same issue on both my Mac (Sonoma 14.1.1) and @beritgreinke 's (I think she is on Monterey).

I still had the version from 7 Dec on my computer and that populated correctly, so it seems to be a new issue related to this release.

**Comment by benhe0** [comment]
hey, i'm trying to reproduce on a mac and i'm almost certain that the issue comes from the main file being blocked. i think macOS only blocks the main executable in /externals/dist/ when the folder is downloaded as .zip or from the browser in general. never had this problem when i directly cloned the repo. can you unblock it to confirm that the issue comes from the main file not being reached by the patch?

**Comment by federicoVisi** [comment]
I opened `externals/dist/main` manually as described [here](https://support.apple.com/en-gb/guide/mac-help/mh40616/mac) but the problem persists...

**Comment by federicoVisi** [comment]
Tried with the updated release with .dmg file in place of the .zip. It works now, `main` looks like an actual executable, you get a security warning when you open the Max patch but you can manually open `main` as described [here](https://support.apple.com/en-gb/guide/mac-help/mh40616/mac) to save it as an exception. Then when you reopen the Max patch everything seems to be working, serial ports appear in the drop-down menu and flashing works fine.
